scripts/keyframes.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python3
+"""Given a gfxreconstruct command file, identify "interesting"
```

Comment:
Not sure a "gfxreconstruct command file" is a thing that exists. On first pass this script seemed completely independent of GFXReconstruct as a whole and gfxrecon-convert in particular.
Shouldn't this read, _"Given a text file of Vulkan function names, one per line, in order of execution, ..."_?

Reply:
That's a better description, thanks.